Задание_№9/Задание_№9/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Задание__9
+{
+    class Program
+    {
+        static void Main()
+        {
+            int[] mas = new int[10];
+            int a, d;
+            Console.WriteLine("Введите первое значение прогресси");
+            a = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Введите разность прогресси");
+            d = Convert.ToInt32(Console.ReadLine());
+            nomber(ref mas, a, d);
+            shouw(mas);
+            Console.WriteLine("\nВведите переменную k");
+            int k;
+            k = Convert.ToInt32(Console.ReadLine());
+            klass(ref mas, k);
+            shouw(mas);
+            rtr(mas);
+            Console.WriteLine("Введите стпень");
+            int n;
+            n = Convert.ToInt32(Console.ReadLine());
+            step(n, mas);
+        }
+        static void nomber(ref int[] mas, int a, int d)
+        {
+
+            for (int i = 0; i < mas.Length; i++)
+            {
+
+                mas[i] = a + i * d;
+
+
+            }
+
+        }
```

Comment:
Методы лучше называть с большой буквы и имя метода начинать с глагола. Также придумывайте значащие имена. Массив в метод и так передается по ссылке, так что передавать ссылку на ссылку не нужно.
```suggestion
        static void FillArray(int[] mas, int a, int d)
        {
            for (int i = 0; i < mas.Length; i++)
                mas[i] = a + i * d;
        }
```